src/Bowling.js

```diff
@@ -0,0 +1,92 @@
+'use srict'
+
+// var Frame = require('../src/frame')
+function Frame (first, second) {
+  this.rolls = [first, second]
```

Comment:
Remember when using prototype it's a good idea to use the _ convention to indicate intended encapsulation